workflow/controller/operator.go

```diff
@@ -1392,6 +1422,32 @@ func (woc *wfOperationCtx) executeTemplate(nodeName string, orgTmpl wfv1.Templat
 		woc.updated = true
 	}
 
+	localParams := make(map[string]string)
```

Comment:
why we got so much going on here?

Reply:
This has just been moved up from where it already existed before.